Shared/Prefs.swift

```diff
@@ -0,0 +1,177 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+public struct PrefsKeys {
+    public static let KeyLastRemoteTabSyncTime = "lastRemoteTabSyncTime"
```

Comment:
nit: I'm pretty sure that Swift 3.0 style guide recommends lowerCamelCase constants https://github.com/raywenderlich/swift-style-guide#constants